boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350.yaml

```diff
@@ -2,6 +2,8 @@ identifier: adp_xc7k_ae350
 name: Andes ADP-XC7K AE350
 type: mcu
 arch: riscv32
+simulation: misc
+simulation_exec: cat
```

Comment:
This is maybe questionable, but it does help to illustrate some things.

I think there is maybe a slightly deeper issue that this is trying to address via workaround though.

Please correct me if I'm wrong, but I think the underlying issue is that it's not possible to have a custom West Runner out of tree? Or perhaps something to do with attaching to a "serial device"?

I know that @nashif is out due to extenuating circumstances right now, but I think he might have some insight into things.